src/Symfony/Component/Notifier/Recipient/EmailRecipient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Recipient;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ *
+ * @experimental in 5.1
+ */
+class EmailRecipient implements EmailRecipientInterface
```

Comment:
If we go down this road, I would prefer to not have an EmailRecipient and SmsRecipient class, but traits instead that you can compose in your own Recipent class.

Reply:
Oh yes, will be much more usable like this. Will change this.